src/systems/systems/src/landing_gear/mod.rs

```diff
@@ -1262,6 +1294,23 @@ impl GearSystemStateMachine {
         true
     }
 
+    /// Checks if sensors are consistent with current state so we can monitor this continuously
+    fn check_gear_state(&mut self, lgciu: &(impl LgciuGearExtension + LgciuDoorPosition)) -> bool {
+        if self.doors_in_transition(lgciu) && self.gears_in_transition(lgciu) {
+            return false;
+        }
+
+        match self.gears_state {
+            GearSystemState::AllUpLocked => {
+                lgciu.all_up_and_locked() && lgciu.all_closed_and_locked()
+            }
+            GearSystemState::AllDownLocked => {
+                lgciu.all_down_and_locked() && lgciu.all_closed_and_locked()
+            }
+            _ => true,
+        }
```

Comment:
Explicitly declare the remaining patterns instead of using _. This is a rule in the code guidelines.

Reply:
Oopsie!